Updown.io-Guide.md

```
@@ -0,0 +1,279 @@
+      "name": "Region 2 - EU West",
+      "hostname": "provider2.akash.network",
+      "location": "eu-west",
+      "port": 443
+    }
+  ]
+}
+```
+
+### Step 2: Configure Updown.io
+
+1. Sign up at https://updown.io if you haven't already
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)